src/frontend/src/test-e2e/flows.ts

```diff
@@ -93,7 +93,7 @@ export const FLOWS = {
     await welcomeView.login();
     await welcomeView.typeUserNumber(userNumber);
     await browser.$("button[data-action='continue']").click();
-    // NOTE: handle recovery nag because there is no recovery phrase
+    // This flows assumes no recovery phrase, so we handle recovery nag here
```

Comment:
Is it a typo `"this flows"`? Shouldn't it be singular?

Can you change the verb "handle" for "skip"? I found the "handle" part to be vague and confusing.

Reply:
Sure, I was a bit too quick with this one. 🙃